### DISCUSSION TITLE
Using V10 without css @layer feature ?

### DISCUSSION THREAD

**Discussion by jerlam06** [primefaces/community#539] [PrimeReact] [answered]
Is there any way to use primereact v10 without using this css @layer feature ? My company's app is huge, we have a bunch of css/scss files imported, and we do not use any @layer stuff. I tried implementing it, it mostly works but create styling issues because of specificity.

**Comment by melloware** [comment, marked as answer]
Already open tickets about this:

https://github.com/primefaces/primereact/issues/5362

https://github.com/primefaces/primereact/issues/5124

**Comment by mertsincan** [comment]
Hi,
We are working on a new styled structure. This will be optional when we complete it. For now, closed.
